src/core/unit.h

```diff
@@ -1046,7 +1046,7 @@ void unit_next_freezer_state(Unit *u, FreezerAction action, FreezerState *ret_ne
 void unit_set_freezer_state(Unit *u, FreezerState state);
 void unit_freezer_complete(Unit *u, FreezerState kernel_state);
 
-int unit_can_live_mount(const Unit *u, sd_bus_error *error);
+bool unit_can_live_mount(const Unit *u, sd_bus_error *error);
```

Comment:
hmm, this is weird. on one hand you are no longer returning an error (i.e. ret val is not bool, doesn't take negative errno anymore), but on the other hand you return an error (via the dbus error param).

I think we should either return an error both ways, or neither way. Given we want an error in some csaes when we call it, i think we should continue `int` as before, and then let the caller choose whether it uses it or not.

Reply:
> > can_live_mount should really just return a bool

> yeah, props should never fail nor log


My dudes, you are literally killing me